openedx/core/djangoapps/bookmarks/views.py

```diff
@@ -314,7 +309,7 @@ def get_usage_key_or_error_response(self, usage_id):
             log.error(error_message)
             return self.error_response(error_message, error_status=status.HTTP_404_NOT_FOUND)
 
-    @swagger_auto_schema(
+    @apidocs.schema(
```

Comment:
While you're at it, why not update this to your new docstring-based style too? :)

Reply:
:+1: